### ISSUE TITLE
pLink 2.3.9 does not want to start the search

### ISSUE THREAD

**Issue by ChengkangLi** [pFindStudio/pLink2#67]
Dear pLink,

I received the email about updating 2.3.8 to 2.3.9 to avoid the found bugs, however, my pLink 2.3.9 does not seem to work either. Search cannot be started, while the status kept staying at "Search Engine is ready to search". I tried several times, however, problem still existed. Can you help me out, please. Thank you.
![image](https://user-images.githubusercontent.com/48132179/71971243-81fb1900-320a-11ea-83c1-995718d094df.png)

Best regards,
Chengkang

**Comment by 01joy** [comment]
Dear @ChengkangLi ,

Will [this issue](https://github.com/pFindStudio/pLink2/issues/31) help you?

**Comment by ChengkangLi** [comment]
Dear @01joy ,

I had tried to remove my selected modification for the search, however, it was still not happy with my setting. I tried to do the simply disulfide search, which worked fine, but none of my personal customise linkers is working in my pLink 2.3.9. I had tried all of my linkers in v2.3.5, and i had noticed that the link.ini format is slightly different. I copied my linkers from the older version to the v2.3.9 with my adjustment, in order to make them work well, and no error popped out when i open my pConfig and the linkers. 

Best regards,
Chengkang
